decidim-core/db/migrate/20170713131308_migrate_user_roles_to_participatory_process_roles.rb

```diff
@@ -8,7 +8,7 @@ def up
       processes = participatory_processes.select { |process| process.organization == user.organization }
       processes.each do |process|
         user.roles.each do |role|
-          Decidim::ParticipatoryProcessUserRole.find_or_create_by(user: user, process: process, role: role)
+          Decidim::ParticipatoryProcessUserRole.find_or_create_by(user: user, participatory_process: process, role: role)
```

Comment:
Agreed. If application specific models were ever to be used (because plain SQL migration is too complicated), they should be defined inline in the migration.

```ruby
 class MigrateUserRolesToParticipatoryProcessRoles < ActiveRecord::Migration[5.1]
  class Decidim::ParticipatoryProcessUserRole < ApplicationRecord
  end

  # ... migration code ...
end

```